console/src/components/Services/RemoteSchema/Permissions/RSPInput.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import { GraphQLInputField } from 'graphql';
+import Pen from './Pen';
+import { useDebouncedEffect } from '../../../../hooks/useDebounceEffect';
+import { isNumberString } from '../../../Common/utils/jsUtils';
+
+interface RSPInputProps {
+  k: string;
+  editMode: boolean;
+  value: string | Record<string, any>;
+  v: GraphQLInputField;
+  setArgVal: (v: Record<string, unknown>) => void;
+  setEditMode: (b: boolean) => void;
+}
+const RSPInput: React.FC<RSPInputProps> = ({
+  k,
+  editMode,
+  value,
+  setArgVal,
+  v,
+  setEditMode,
+}) => {
+  const [localValue, setLocalValue] = useState<string>(value as string);
```

Comment:
if `value` can be either `string` or `Record<string, any>` why do we do unsafe type assertion here?